x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/cases_webhook/action_variables.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ActionVariable } from '@kbn/alerting-plugin/common';
+import * as i18n from './translations';
+
+export const casesVars: ActionVariable[] = [
+  { name: 'case.title', description: i18n.CASE_TITLE_DESC, useWithTripleBracesInTemplates: true },
+  {
+    name: 'case.description',
+    description: i18n.CASE_DESCRIPTION_DESC,
+    useWithTripleBracesInTemplates: true,
```

Comment:
triple braces in mustache replaces raw values while double braces does an html escape. triple is meant to be used with replacing JSON values like we are doing 👍